network.go

```diff
@@ -272,10 +272,17 @@ func (s *sandbox) updateInterface(netHandle *netlink.Handle, iface *types.Interf
 		// Find the interface link from its hardware address.
 		link, err = linkByHwAddr(netHandle, iface.HwAddr)
 		if err != nil {
-			return nil, grpcStatus.Errorf(codes.Internal, "updateInterface: %v", err)
+			return nil, grpcStatus.Errorf(codes.Internal, "updateInterface by HwAddr: %v", err)
```

Comment:
I suggest you could do this:
```
if iface.PciAddr != "" {
     name , err := getPCIDeviceName(s, iface.PciAddr)
      // Get link based on name
      link, err = netHandle.LinkByName(name) 
      .....
} else if if iface.HwAddr != "" {
 ....
}

Reply:
@amshinde ok, thanks for the suggestions